### DISCUSSION TITLE
PyMuPDF branch for 1.20.0 release candidates

### DISCUSSION THREAD

**Discussion by julian-smith-artifex-com** [pymupdf/PyMuPDF#1753] [Announcements]
I've just pushed some commits to a new branch `1.20.0rc`.

This branch will hopefully be copied onto the master branch very soon for an official release of PyMuPDF-1.20.0.

The branch contains some changes to PyMuPDF that allow it to work with the recently-released MuPDF-1.20.0.

It also has some changes to the `setup.py` script to simplify some aspects of how we build wheels for PyMuPDF. In particular, `setup.py` now contains a hard-coded URL for the specific release of MuPDF that we require. By default we always use the MuPDF from this URL when creating an sdist or building PyMuPDF (e.g. into a wheel). See comments at top of `setup.py` for more information.

The `1.20.0rc` branch has a release candidate tag `1.20.0rc1`. Wheels for this have been uploaded to pypi.org and can be installed with`pip install pymupdf==1.20.0rc1` (pypi will continue to install PyMuPDF-1.19.6 by default until we release 1.20.0).

Thanks,

\- Julian

**Comment by jamie-lemon** [comment]
Excellent 👍
